lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -53,8 +60,23 @@ def get(self, request, course_key_string):
         course_key = CourseKey.from_string(course_key_string)
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
+        blocks = get_course_date_blocks(course, request.user, request, num_assignments=1)
+
+        # User locale settings
+        user_timezone_locale = user_timezone_locale_prefs(request)
+        user_timezone = user_timezone_locale['user_timezone']
+
+        dates_tab_link = reverse('dates', args=[course.id])
+        if course_home_mfe_dates_tab_is_active(course.id):
+            return get_microfrontend_url(course_key=course.id, view_name='dates')
+
+
```

Comment:
This will give a quality error for having two spaces here. I can show you how to check what quality errors are showing up as part of your PR :)